roachtest: gorm failed

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8997643?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8997643?buildTab=artifacts#/gorm) on master @ [d4a584e49f0b1ca89738376090939d7669c3b3db](https://github.com/cockroachdb/cockroach/commits/d4a584e49f0b1ca89738376090939d7669c3b3db):


```
test artifacts and logs in: /artifacts/gorm/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.6-dev-d4a584e49f0b1ca89738376090939d7669c3b3db
Tests run against gorm v1.24.6
1 Total Tests Run
0 tests passed
1 test failed
0 tests skipped
0 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.[build failed] - unknown (unexpected)
For a full summary look at the gorm artifacts
An updated blocklist (gormBlocklist) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #94584 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
/cc @cockroachdb/sql-sessions
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-25226

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9011803?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9011803?buildTab=artifacts#/gorm) on master @ [5b2a5670cbbe895d76602c230390816e783e0caa](https://github.com/cockroachdb/cockroach/commits/5b2a5670cbbe895d76602c230390816e783e0caa):


```
test artifacts and logs in: /artifacts/gorm/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.6-dev-5b2a5670cbbe895d76602c230390816e783e0caa
Tests run against gorm v1.24.6
1 Total Tests Run
0 tests passed
1 test failed
0 tests skipped
0 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.[build failed] - unknown (unexpected)
For a full summary look at the gorm artifacts
An updated blocklist (gormBlocklist) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #98430 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.2 release-blocker]
- #94584 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9029456?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9029456?buildTab=artifacts#/gorm) on master @ [e4924e2b9be4a36d466beab53a80df9241df4783](https://github.com/cockroachdb/cockroach/commits/e4924e2b9be4a36d466beab53a80df9241df4783):


```
test artifacts and logs in: /artifacts/gorm/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.6-dev-e4924e2b9be4a36d466beab53a80df9241df4783
Tests run against gorm v1.24.6
1 Total Tests Run
0 tests passed
1 test failed
0 tests skipped
0 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.[build failed] - unknown (unexpected)
For a full summary look at the gorm artifacts
An updated blocklist (gormBlocklist) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #98430 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.2 release-blocker]
- #94584 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9050605?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9050605?buildTab=artifacts#/gorm) on master @ [10c1e3e01b7da4cced0478e1bfd711a1c9be9afc](https://github.com/cockroachdb/cockroach/commits/10c1e3e01b7da4cced0478e1bfd711a1c9be9afc):


```
test artifacts and logs in: /artifacts/gorm/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.7-dev-10c1e3e01b7da4cced0478e1bfd711a1c9be9afc
Tests run against gorm v1.24.6
1 Total Tests Run
0 tests passed
1 test failed
0 tests skipped
0 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.[build failed] - unknown (unexpected)
For a full summary look at the gorm artifacts
An updated blocklist (gormBlocklist) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #98430 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.2 release-blocker]
- #94584 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9070587?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9070587?buildTab=artifacts#/gorm) on master @ [9b43dc50d468171bc9919dc684b7274b63e13da2](https://github.com/cockroachdb/cockroach/commits/9b43dc50d468171bc9919dc684b7274b63e13da2):


```
test artifacts and logs in: /artifacts/gorm/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.7-dev-9b43dc50d468171bc9919dc684b7274b63e13da2
Tests run against gorm v1.24.6
1 Total Tests Run
0 tests passed
1 test failed
0 tests skipped
0 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.[build failed] - unknown (unexpected)
For a full summary look at the gorm artifacts
An updated blocklist (gormBlocklist) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #98430 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.2 release-blocker]
- #94584 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9088337?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9088337?buildTab=artifacts#/gorm) on master @ [d68f5cdfb25476aaf874bff0bb6515eba02efcd2](https://github.com/cockroachdb/cockroach/commits/d68f5cdfb25476aaf874bff0bb6515eba02efcd2):


```
test artifacts and logs in: /artifacts/gorm/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.7-dev-d68f5cdfb25476aaf874bff0bb6515eba02efcd2
Tests run against gorm v1.24.6
1 Total Tests Run
0 tests passed
1 test failed
0 tests skipped
0 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.[build failed] - unknown (unexpected)
For a full summary look at the gorm artifacts
An updated blocklist (gormBlocklist) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #98430 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.2]
- #94584 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9106835?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9106835?buildTab=artifacts#/gorm) on master @ [f0489334a0ee6980a9d365b361d2fce4b2cdc05b](https://github.com/cockroachdb/cockroach/commits/f0489334a0ee6980a9d365b361d2fce4b2cdc05b):


```
test artifacts and logs in: /artifacts/gorm/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.7-dev-f0489334a0ee6980a9d365b361d2fce4b2cdc05b
Tests run against gorm v1.24.6
1 Total Tests Run
0 tests passed
1 test failed
0 tests skipped
0 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.[build failed] - unknown (unexpected)
For a full summary look at the gorm artifacts
An updated blocklist (gormBlocklist) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #98430 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.2]
- #94584 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9121674?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9121674?buildTab=artifacts#/gorm) on master @ [6c99966f604f3521acdb925b9f689529ffd46df3](https://github.com/cockroachdb/cockroach/commits/6c99966f604f3521acdb925b9f689529ffd46df3):


```
test artifacts and logs in: /artifacts/gorm/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.7-dev-6c99966f604f3521acdb925b9f689529ffd46df3
Tests run against gorm v1.24.6
1 Total Tests Run
0 tests passed
1 test failed
0 tests skipped
0 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.[build failed] - unknown (unexpected)
For a full summary look at the gorm artifacts
An updated blocklist (gormBlocklist) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #98430 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.2]
- #94584 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9130968?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9130968?buildTab=artifacts#/gorm) on master @ [53dbb86acb1d48309530181b94838faf937084d3](https://github.com/cockroachdb/cockroach/commits/53dbb86acb1d48309530181b94838faf937084d3):


```
test artifacts and logs in: /artifacts/gorm/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.7-dev-53dbb86acb1d48309530181b94838faf937084d3
Tests run against gorm v1.24.6
1 Total Tests Run
0 tests passed
1 test failed
0 tests skipped
0 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.[build failed] - unknown (unexpected)
For a full summary look at the gorm artifacts
An updated blocklist (gormBlocklist) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #98430 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.2]
- #94584 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9182736?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9182736?buildTab=artifacts#/gorm) on master @ [78bb7696847e1d8abf5d6a199c6dd3414a568691](https://github.com/cockroachdb/cockroach/commits/78bb7696847e1d8abf5d6a199c6dd3414a568691):


```
test artifacts and logs in: /artifacts/gorm/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.7-dev-78bb7696847e1d8abf5d6a199c6dd3414a568691
Tests run against gorm v1.24.6
1 Total Tests Run
0 tests passed
1 test failed
0 tests skipped
0 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.[build failed] - unknown (unexpected)
For a full summary look at the gorm artifacts
An updated blocklist (gormBlocklist) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99199 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1 release-blocker]
- #98430 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.2]
- #94584 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9205663?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9205663?buildTab=artifacts#/gorm) on master @ [5493fdfec4e1762c4502fb2f5d42fd28292c9c9d](https://github.com/cockroachdb/cockroach/commits/5493fdfec4e1762c4502fb2f5d42fd28292c9c9d):


```
test artifacts and logs in: /artifacts/gorm/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.7-dev-5493fdfec4e1762c4502fb2f5d42fd28292c9c9d
Tests run against gorm v1.24.6
1 Total Tests Run
0 tests passed
1 test failed
0 tests skipped
0 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.[build failed] - unknown (unexpected)
For a full summary look at the gorm artifacts
An updated blocklist (gormBlocklist) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99199 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1 release-blocker]
- #98430 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.2]
- #94584 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9225780?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9225780?buildTab=artifacts#/gorm) on master @ [19a6b804d3aff74d74619a75cac3b52338c7aa02](https://github.com/cockroachdb/cockroach/commits/19a6b804d3aff74d74619a75cac3b52338c7aa02):


```
test artifacts and logs in: /artifacts/gorm/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-19a6b804d3aff74d74619a75cac3b52338c7aa02
Tests run against gorm v1.24.6
1 Total Tests Run
0 tests passed
1 test failed
0 tests skipped
0 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.[build failed] - unknown (unexpected)
For a full summary look at the gorm artifacts
An updated blocklist (gormBlocklist) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99199 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1 release-blocker]
- #98430 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.2]
- #94584 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9244147?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9244147?buildTab=artifacts#/gorm) on master @ [0fcc33bc2870961b9387999d8a9fed97fccbb2ae](https://github.com/cockroachdb/cockroach/commits/0fcc33bc2870961b9387999d8a9fed97fccbb2ae):


```
test artifacts and logs in: /artifacts/gorm/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-0fcc33bc2870961b9387999d8a9fed97fccbb2ae
Tests run against gorm v1.24.6
1 Total Tests Run
0 tests passed
1 test failed
0 tests skipped
0 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.[build failed] - unknown (unexpected)
For a full summary look at the gorm artifacts
An updated blocklist (gormBlocklist) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99199 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #98430 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.2]
- #94584 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9253372?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9253372?buildTab=artifacts#/gorm) on master @ [d107217dac5d817cc115bc0e97b7e53c0f2878bf](https://github.com/cockroachdb/cockroach/commits/d107217dac5d817cc115bc0e97b7e53c0f2878bf):


```
test artifacts and logs in: /artifacts/gorm/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-d107217dac5d817cc115bc0e97b7e53c0f2878bf
Tests run against gorm v1.24.6
1 Total Tests Run
0 tests passed
1 test failed
0 tests skipped
0 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.[build failed] - unknown (unexpected)
For a full summary look at the gorm artifacts
An updated blocklist (gormBlocklist) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99199 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #98430 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.2]
- #94584 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9262923?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9262923?buildTab=artifacts#/gorm) on master @ [2bd2c806ab3044569b09e0a205b5bc0452ad4e2b](https://github.com/cockroachdb/cockroach/commits/2bd2c806ab3044569b09e0a205b5bc0452ad4e2b):


```
test artifacts and logs in: /artifacts/gorm/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-2bd2c806ab3044569b09e0a205b5bc0452ad4e2b
Tests run against gorm v1.24.6
1 Total Tests Run
0 tests passed
1 test failed
0 tests skipped
0 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.[build failed] - unknown (unexpected)
For a full summary look at the gorm artifacts
An updated blocklist (gormBlocklist) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99199 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #98430 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.2]
- #94584 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9306404?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9306404?buildTab=artifacts#/gorm) on master @ [bb5c97b5d2073a27e7ec88654b05ac913112f610](https://github.com/cockroachdb/cockroach/commits/bb5c97b5d2073a27e7ec88654b05ac913112f610):


```
test artifacts and logs in: /artifacts/gorm/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-bb5c97b5d2073a27e7ec88654b05ac913112f610
Tests run against gorm v1.24.6
1 Total Tests Run
0 tests passed
1 test failed
0 tests skipped
0 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.[build failed] - unknown (unexpected)
For a full summary look at the gorm artifacts
An updated blocklist (gormBlocklist) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99199 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #98430 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.2]
- #94584 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9329887?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9329887?buildTab=artifacts#/gorm) on master @ [1f8024bf14433ca169e5a8c3768c5d223dc5018c](https://github.com/cockroachdb/cockroach/commits/1f8024bf14433ca169e5a8c3768c5d223dc5018c):


```
test artifacts and logs in: /artifacts/gorm/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-1f8024bf14433ca169e5a8c3768c5d223dc5018c
Tests run against gorm v1.24.6
1 Total Tests Run
0 tests passed
1 test failed
0 tests skipped
0 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.[build failed] - unknown (unexpected)
For a full summary look at the gorm artifacts
An updated blocklist (gormBlocklist) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99199 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #98430 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.2]
- #94584 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9353426?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9353426?buildTab=artifacts#/gorm) on master @ [41581c367b3a1d7893c2945465c57ffa841035e3](https://github.com/cockroachdb/cockroach/commits/41581c367b3a1d7893c2945465c57ffa841035e3):


```
test artifacts and logs in: /artifacts/gorm/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-41581c367b3a1d7893c2945465c57ffa841035e3
Tests run against gorm v1.24.6
1 Total Tests Run
0 tests passed
1 test failed
0 tests skipped
0 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.[build failed] - unknown (unexpected)
For a full summary look at the gorm artifacts
An updated blocklist (gormBlocklist) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99199 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #98430 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.2]
- #94584 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9353437?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9353437?buildTab=artifacts#/gorm) on master @ [41581c367b3a1d7893c2945465c57ffa841035e3](https://github.com/cockroachdb/cockroach/commits/41581c367b3a1d7893c2945465c57ffa841035e3):


```
test artifacts and logs in: /artifacts/gorm/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-41581c367b3a1d7893c2945465c57ffa841035e3
Tests run against gorm v1.24.6
1 Total Tests Run
0 tests passed
1 test failed
0 tests skipped
0 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.[build failed] - unknown (unexpected)
For a full summary look at the gorm artifacts
An updated blocklist (gormBlocklist) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99199 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #98430 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.2]
- #94584 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/gorm.go
+++ b/pkg/cmd/roachtest/tests/gorm.go
@@ -104,7 +104,7 @@ func registerGORM(r registry.Registry) {
 		err = c.RunE(
 			ctx,
 			node,
-			fmt.Sprintf(`cd %s && go get -u -t ./... && go mod download && go mod tidy `, gormTestPath),
+			fmt.Sprintf(`cd %s && go mod tidy && go mod download`, gormTestPath),
 		)
 		require.NoError(t, err)
 

